test/model/src/include/model/driver/kv_workload.h

```diff
@@ -0,0 +1,332 @@
+ *     A representation of this workload operation.
+ */
+struct rollback_to_stable {
+
+    /*
+     * rollback_to_stable::rollback_to_stable --
+     *     Create the operation.
+     */
+    inline rollback_to_stable() {}
+};
+
+/*
+ * rollback_transaction --
+ *     A representation of this workload operation.
+ */
+struct rollback_transaction {
+    txn_id_t txn_id;
+
+    /*
+     * rollback_transaction::rollback_transaction --
+     *     Create the operation.
+     */
+    inline rollback_transaction(txn_id_t txn_id) : txn_id(txn_id) {}
+};
+
+/*
+ * set_commit_timestamp --
+ *     A representation of this workload operation.
+ */
+struct set_commit_timestamp {
+    txn_id_t txn_id;
+    timestamp_t commit_timestamp;
+
+    /*
+     * set_commit_timestamp::set_commit_timestamp --
+     *     Create the operation.
+     */
+    inline set_commit_timestamp(txn_id_t txn_id, timestamp_t commit_timestamp)
+        : txn_id(txn_id), commit_timestamp(commit_timestamp)
+    {
+    }
+};
+
+/*
+ * set_stable_timestamp --
+ *     A representation of this workload operation.
+ */
+struct set_stable_timestamp {
+    timestamp_t stable_timestamp;
+
+    /*
+     * set_stable_timestamp::set_stable_timestamp --
+     *     Create the operation.
+     */
+    inline set_stable_timestamp(timestamp_t stable_timestamp) : stable_timestamp(stable_timestamp)
+    {
+    }
+};
+
+/*
+ * truncate --
+ *     A representation of this workload operation.
+ */
+struct truncate {
+    table_id_t table_id;
+    txn_id_t txn_id;
+    data_value start;
+    data_value stop;
+
+    /*
+     * truncate::truncate --
+     *     Create the operation.
+     */
+    inline truncate(
+      table_id_t table_id, txn_id_t txn_id, const data_value &start, const data_value &stop)
+        : table_id(table_id), txn_id(txn_id), start(start), stop(stop)
+    {
+    }
+};
+
+/*
+ * any --
+ *     Any workload operation.
+ */
+using any = std::variant<begin_transaction, checkpoint, commit_transaction, create_table, insert,
+  prepare_transaction, remove, restart, rollback_to_stable, rollback_transaction,
+  set_commit_timestamp, set_stable_timestamp, truncate>;
+
+} /* namespace operation */
+
+/*
+ * kv_workload --
+ *     A workload representation for a key-value database.
+ */
+class kv_workload {
+
+public:
+    /*
+     * kv_workload::kv_workload --
+     *     Create a new workload.
+     */
+    inline kv_workload() {}
+
+    /*
+     * kv_workload::operator<< --
+     *     Add an operation to the workload.
+     */
+    inline kv_workload &
+    operator<<(const operation::any &op)
+    {
+        _operations.push_back(op);
+        return *this;
+    }
+
+    /*
+     * kv_workload::operator<< --
+     *     Add an operation to the workload.
+     */
+    inline kv_workload &
+    operator<<(operation::any &&op)
+    {
+        _operations.push_back(std::move(op));
+        return *this;
+    }
+
+    /*
+     * kv_workload::run --
+     *     Run the workload.
```

Comment:
Change to "Run the workload in the model."

Reply:
Fixed.